### PR TITLE
Add additional parameters to Camera and support xiTypeInteger64

### PR DESCRIPTION
**Pull request by lgarbarini**
Adds some additional parameters, one, `Timestamp` requires `u64` type, so also add support for that

### CODE REVIEW

File: src/camera.rs [hunk 1]
```diff
@@ -603,6 +642,15 @@ impl Camera {
 
         /// Recent Frame mode.
         mut recent_frame: XI_SWITCH::Type;
+
+        /// Setting the API debug level allows to select amount of messages stored to debug output.
+        mut debug_level: XI_DEBUG_LEVEL::Type;
```
Comment (superbort) [inline src/camera.rs]:
```suggestion
/// Removed
```
I would not add this parameter to the camera and here is why:

You can already set the debug level using the static top level xiapi::set_debug_level function. The debug level in xiAPI is always global, so it does not make sense to have it on the camera. The static function has the additional advantage that you can use it before even openining a camera, so you can get debug output from the beginning.

File: src/camera.rs [hunk 2]
```diff
@@ -603,6 +642,15 @@ impl Camera {
...
+
+        /// Defines the size of the acquisition buffer in bytes. This is a circle buffer which contains image data from sensor.
+        mut acq_buffer_size: i32;
```
Comment (superbort) [inline src/camera.rs]:
```suggestion
        /// Removed
```
I agree that this functionality is missing, but unfortunately just adding the parameter is not a good solution. ACQ_BUFFER_SIZE is a 32-bit signed integer in the C API. The highest possible value would only get us to 2GB, which may be too small in a real-world application.
In the C API this is solved by introducing an additional parameter ACQ_BUFFER_SIZE_UNIT and if you really want to use this directly in Rust, you can do so using xiapi_sys. For these higher level bindings however I would suggest wrapping both of these parameters in a nicer function that also catches some of the potential errors (e.g. negative parameter values). 
I will deal with this in a separate ticket.

File: src/camera.rs [hunk 3]
```diff
@@ -603,6 +642,15 @@ impl Camera {
...
+
+        /// Configures image data delivery target to CPU RAM (default) or GPU RAM.
+        mut transport_data_target: XI_TRANSPORT_DATA_TARGET_MODE::Type;
```
Comment (superbort) [inline src/camera.rs:647]: Did you test this with any other target? Using the GPU may require some additional changes, but this is a good start.

**Review by superbort — CHANGES_REQUESTED** [review]
I suggested some changes, mostly to prevent the shortcomings of the C API to translate directly to Rust. We can do better than that.

File: src/camera.rs (hunk 2 above)
Comment (superbort) [reply src/camera.rs]: #16 created for this.

File: src/camera.rs (hunk 1 above)
Comment (lgarbarini) [reply src/camera.rs]: Wow, somehow I missed `set_debug_level`! Updated

File: src/camera.rs (hunk 3 above)
Comment (lgarbarini) [reply src/camera.rs:647]: I have tested this with both `UNIFIED` and `ZEROCOPY` and it seems to just work. I don't have the proper setup to test GPU Direct

**Review by superbort — APPROVED** [review]

**Pull request merged**